commit 0ed159ec438156289ad94b8bf0db9d617ca8c87d (refs/changes/02/3702/3)
Author: John Levon <john.levon@joyent.com>
Date:   2018-03-26T19:23:02+00:00 (1 year, 6 months ago)
    
    OS-6800 bhyve-opts should be bhyve-extra-opts
    Reviewed by: Mike Gerdts <mike.gerdts@joyent.com>
    Approved by: Mike Gerdts <mike.gerdts@joyent.com>

diff --git a/src/vm/node_modules/VM.js b/src/vm/node_modules/VM.js
index f6679e14..202195d6 100644
--- a/src/vm/node_modules/VM.js
+++ b/src/vm/node_modules/VM.js
@@ -6086,7 +6086,7 @@ function buildZonecfgUpdate(vmobj, payload, log)
         setAttr('com1', 'com1');
         setAttr('com2', 'com2');
 
-        setAttr('bhyve-opts', 'bhyve_opts');
+        setAttr('bhyve-extra-opts', 'bhyve_extra_opts');
     }
 
     // NOTE: Thanks to normalizePayload() we'll only have these when relevant
@@ -11997,7 +11997,6 @@ exports.start = function (uuid, extra, options, callback)
             'nics',
             'platform_buildstamp',
             'bhyve_extra_opts',
-            'bhyve_opts',
             'ram',
             'resolvers',
             'state',
diff --git a/src/vm/node_modules/proptable.js b/src/vm/node_modules/proptable.js
index 7bbb6c5e..fd698311 100644
--- a/src/vm/node_modules/proptable.js
+++ b/src/vm/node_modules/proptable.js
@@ -325,14 +325,14 @@ exports.properties = {
             type: 'boolean'
         },
         zonexml: 'zone.autoboot'
-    }, bhyve_opts: {
+    }, bhyve_extra_opts: {
         payload: {
             allowed: {
                 'bhyve': ['create', 'receive', 'update']
             },
             type: 'string'
         },
-        zonexml: 'zone.attr.bhyve-opts'
+        zonexml: 'zone.attr.bhyve-extra-opts'
     }, billing_id: {
         payload: {
             allowed: {
